### ISSUE TITLE
plugins.xml not placed in project tree and reference to DroidGap missing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#39]
```
What steps will reproduce the problem?
1. Using Ubuntu 11.04, Eclipse Helios Service Release 2 Build id: 
20110301-1815, and PhonegapForAndroid 1.2.3.301105010956, create a new PhoneGap 
project using the wizard.

What is the expected output? What do you see instead?
plugins.xml should be placed in the /res/xml directory of the project and a 
reference to the DroidGap Activity should be inserted in AndroidManifest.xml.

Both are missing.

```

Original issue reported on code.google.com by `jackch...@gmail.com` on 9 Aug 2011 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
Yes, AppLaud needs to have an update to handle the PhoneGap 1.0 configuration 
options. Until then, you need to do it manually. I hope to make the update in 
the next few weeks. Star this bug or watch the mailing list for updates.
```

Original comment by `paul.beu...@gmail.com` on 10 Aug 2011 at 5:15
- Changed state: **Accepted**
- Added labels: **Milestone-Release1.2.4**

**Comment by GoogleCodeExporter** [comment]
```
Coming in a few days.  Preview release available at 
http://svn.codespot.com/a/eclipselabs.org/mobile-web-development-with-phonegap/t
runk/download
```

Original comment by `paul.beu...@gmail.com` on 16 Aug 2011 at 11:51

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2.4
```

Original comment by `paul.beu...@gmail.com` on 19 Aug 2011 at 4:30
- Changed state: **Fixed**
